src/routes/houses.routes.js

```diff
@@ -1,14 +1,23 @@
-import createNewHouseController from '../controllers/house.controllers';
+import { createNewHouseController, getHouseDetails } from '../controllers/house.controllers';
+
 const { StatusCodes } = require('http-status-codes');
 
-const createNewHouseRoutes = (router) => {
+const housesRoutes = (router) => {
   router.post('/create-new-house', async (req, res) => {
     const response = await createNewHouseController(req.body);
     if (!response || !response.status)
       return res.status(StatusCodes.INTERNAL_SERVER_ERROR).json({ status: 'server error' });
     if (response.status === 'invalid') return res.status(StatusCodes.BAD_REQUEST).json(response);
-    res.status(StatusCodes.CREATED).json(response);
+    return res.status(StatusCodes.CREATED).json(response);
+  });
+
+  router.get('/house/:id', async (req, res) => {
+    const response = await getHouseDetails(req.params.id);
```

Comment:
You could possibly consider some kind of abstraction for the code below as it's very similar to the code for the route `create-new-house`. You don't have to create it yourself as Tomek prepared something that could probably also be used here.